src/main/java/pl/tomaszgronski/projektksiegowy/invoice/db/FileHelper.java

```diff
@@ -24,6 +27,8 @@
     public List<Invoice> getAllDataFromFile() {
         LOGGER.info("Get all invoices");
         ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
you can use one objectMapper, or even better - have it injected by Spring. Don't create objectMapper in every method call